test/linear_solver_routines/krylov_helpers/proc_arnoldi.jl

```diff
@@ -0,0 +1,80 @@
+# Date: 12/03/2024
+# Author: Christian
+# Purpose: Test the arnoldi iteration
+
+module ProceduralArnoldi
+
+using Test, RLinearAlgebra, LinearAlgebra, Random
+
+@testset "Arnoldi Iteration -- Procedural" begin
+
+    Random.seed!(1010)
+
+    #####################
+    # test definition
+    #####################
+    @test isdefined(RLinearAlgebra, :arnoldi)
+
+    #####################
+    # test errors
+    #####################
+    
+    # not square
+    nrow = 100
+    ncol = 50
```

Comment:
Either move these somewhere above to indicate that they are test parameters, or use the `let` block